ResourceStatusSystem/Command/RemainingDiskSpaceCommand.py

```diff
@@ -0,0 +1,102 @@
+''' RemainingDiskSpaceCommand
+
+  The Command gets the remaining space that is left in a DIRAC Storage Element
+
+'''
+
+from datetime                                                   import datetime
+from DIRAC                                                      import S_OK, S_ERROR, gConfig
+from DIRAC.ResourceStatusSystem.Command.Command                 import Command
+from DIRAC.Core.DISET.RPCClient                                 import RPCClient
+
+__RCSID__ = '$Id:  $'
+
+
+class RemainingDiskSpaceCommand( Command ):
+  '''
+  Uses diskSpace method to get the remaining space
+  '''
+
+  def __init__( self, args = None ):
+
+    super( RemainingDiskSpaceCommand, self ).__init__( args )
+
+    self.rpc = None
+    self.rsClient = None
+
+  def getUrl(self, SE, protocol = None ):
```

Comment:
instantiate the SE and call getBasePath (or something very simmilar, I don't remember exactly)